lib/Facebook.js

```diff
@@ -407,17 +409,18 @@ function Facebookbot(configuration) {
                 if (cb) { cb(null, facebook_botkit.webserver); }
             });
 
-
-        request.post('https://' + api_host + '/me/subscribed_apps?access_token=' + configuration.access_token,
-            function(err, res, body) {
-                if (err) {
-                    facebook_botkit.log('Could not subscribe to page messages');
-                } else {
-                    facebook_botkit.debug('Successfully subscribed to Facebook events:', body);
-                    facebook_botkit.startTicking();
-                }
-            });
-
+        var access_tokens = (typeof (configuration.access_token) === 'string') ? [configuration.access_token] : configuration.access_token;
+        Object.keys(access_tokens).forEach(function(page) {
```

Comment:
you are right. i should have used { 0: configuration.access_token } instead of [configuration.access_token]. let me correct this right now across the board.